### ISSUE TITLE
Push Notifications Being Sent But Not Received - Confirmed Certs are Working with openssl

### ISSUE THREAD

**Issue by jpgilchrist** [raix/push#130]
Hello @raix,

First and foremost, great package. Instructions were very detailed and I got it up and running on my local. However, I deployed it to my modulus server and ran it on my device. Upon doing so, I ran into the following issues:

1. The logs show "Push: Sent message "Someone stoked the fire!" to 1 ios apps 0 android apps", which is correct based on my query passed to Push.send. However, the notification is never received on the device. 

2. In performing some testing, at one point I deleted the app on my device and redeployed it with "meteor run ios-device --mobile-server {my mobile server} and upon attempting to send a notification, the logs showed, "Push: Sent message "Someone stoked the fire!" to 2 ios apps 0 android apps". There really should only be one device for the query I am using. This lends me to believe that the update token method is not idempotent. Do you have any plans in resolving this?

Cheers,

James Gilchrist.

**Comment by jpgilchrist** [comment]
Can I get some input on this? I moved over to a D.O. droplet and I tested my certs from the server and verified they were good. Still getting the same issues as above. It says they were sent, but aren't being received. Any help would be greatly appreciated.

**Comment by raix** [comment]
@jpgilchrist did you solve the issue?

**Comment by jpgilchrist** [comment]
@raix yes. When I first opened this and #131 I thought they were distinct. Then I closed #131 because I thought it was a Modulus issue (without really looking at it). Then when I encountered the same issue again on digital ocean servers, I opened #133 (probably should have reopened #131). Turns out they were all the same thing.

Sorry for the confusing slew of new issues requests! Haha. 

To recap the solution: set the NODE_ENV variable to development or production to match the value used in the config.push.json file for production: true or production: false.
